### PR TITLE
Describe new way to clone objects and add benchmark to show why.

### PR DESCRIPTION
**Pull request by Xevi**
Show why `JSON.parse(JSON.stringify(object))` is a good alternative of `lang.clone(object)` when trying to clone objects (3x times faster???).
###### Reviewers
- [x] @alvaroortega 
- [x] @basil-conto
- [ ] @floridemai  
- [x] @full-of-foo 
- [ ] @io41
- [ ] @jrochenrfm 
- [ ] @ldoran
- [x] @marzetas
- [ ] @PieceOfMeat

### CODE REVIEW

**Comment by full-of-foo** [comment]
:sheep:

**Comment by alvaroortega** [comment]
:shipit:

**Comment by io41** [comment]
Shouldn't this be a common library function? `CMN.utils.clone()` or something like that?

File: frontend.md [hunk 1]
```diff
@@ -37,7 +37,36 @@ Memory Management [keynote][memory-management-keynote] and [video][memory-manage
...
+      console.log('Fastest is ' + this.filter('fastest').pluck('name'));
+    })
+    .run({ 'async': true });
+});
+```
+> JSON#clone x 416,693 ops/sec ±1.86% (90 runs sampled)
+
+> Dojo#clone x 136,406 ops/sec ±1.97% (85 runs sampled)
+
+> Fastest is JSON#clone
```
Comment (marzetas) [inline frontend.md:68]: While I like reading why to choose one option over another, I think it's a bit too verbose for the guidelines page. Just the first two lines would do the job, imho.

**Comment by basil-conto** [comment]
@io41 Good point; I would be in favour of that.

**Comment by Xevi** [comment]
@io41 really? Do you think is necessary `CMN.utils.clone`? I think it is not really necessary but I'm not against.

File: frontend.md (hunk 1 above)
Comment (full-of-foo) [inline frontend.md:68]: +1, I agree

**Comment by ldoran** [comment]
Closing old PRs

**Pull request closed without merging**